### PR TITLE
Tag cloud resources to sst_conversions

### PR DESCRIPTION
**Pull request by thrix**
We now have cloud costs reporting in Testing Farm.

This is required to tag the cloud resources to this project's sst team.

Will be soon documented here:

https://github.com/packit/packit.dev/pull/632

### CODE REVIEW

File: .packit.yaml [hunk 1]
```diff
@@ -44,10 +44,17 @@ jobs:
...
   identifier: "tier0"
   tmt_plan: "tier0"
+  # Run on Red Testing Farm Hat Ranch, tag resources to sst_conversions
+  use_internal_tf: True
+  tf_extra_params:
+    environments:
+      - settings:
+          provisioning:
+            tags:
+              BusinessUnit: sst_conversions
```
Comment (danmyway) [inline .packit.yaml:57]:
```suggestion
              BusinessUnit: rhel-sst-conversions
```
@thrix is there any convention for the tags from your POV, or should we rather align this with the rover groups?

Comment (danmyway) [reply .packit.yaml:57]: Disscussed this with the team.
@bocekm said:
_> `sst_conversions` is a label that RHEL BU uses in Jira, for example. From what I know there are just RHEL subsystems - I'm not aware of seeing the term subsystem outside of RHEL. So, I'd say `sst_conversions` is fine._

So we can stick with the `sst_conversions`

**Review by danmyway — APPROVED** [review]
LGTM, thanks for this @thrix !

**Comment by danmyway** [comment]
Merging, no need to wait for the pipeline to finish.

File: .packit.yaml (hunk 1 above)
Comment (thrix) [reply .packit.yaml:57]: cool, ty!

**Pull request merged**